webpack.config.js

```diff
@@ -8,15 +8,15 @@ module.exports = {
     path: path.join(__dirname, '/dist'),
     filename: 'bundle.js',
   },
+  resolve: {
+    extensions: ['', '.jsx', '.js', '.json']
+  },
   module: {
     loaders: [
       {
-        test: /\.jsx?$/,
+        test: /(\.js|\.jsx)$/,
```

Comment:
The above regex is equivalent since the ? means 0 or 1, it was the resolve line that was needed.